commands/completion-prompt.ts

```diff
@@ -0,0 +1,28 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+import helpers = require("../helpers");
+
+export class CompletionPromptCommand implements ICommand {
+
+	constructor(private $autoCompletionService: IAutoCompletionService,
+				private $logger: ILogger,
+				private $prompter: IPrompter) {
+	}
+
+	public disableAnalytics = true;
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$logger.out("If you are using bash or zsh, you can enable command line auto completion.");
```

Comment:
Maybe we should notify the user that we will write some data in bash_profile, zsh and bashrc